Makefile

```diff
@@ -19,15 +19,15 @@ test:
 
 .PHONY: install_lint
 install_lint:
-	go get github.com/golang/lint/golint
+	go get golang.org/x/lint/golint
```

Comment:
this is a surprise. They made it experimental?

Reply:
Basically is extensions of standard library. E.g., `x/net` `x/crypto` etc